src/popup-page/modules/postage-batch-element.tsx

```diff
@@ -47,8 +56,8 @@ export function PostageBatchElement(): ReactElement {
 
       return (
         <tr key={postageBatch.batchID + selected} className={selected ? 'bold' : ''}>
-          <td>{postageBatch.batchID}</td>
-          <td>{postageBatch.utilization}</td>
+          <td>{truncatePostageBatchId(postageBatch.batchID)}</td>
+          <td>{calculateUsagePercentage(postageBatch)}%</td>
```

Comment:
I think you need to multiply by 100 and then round it, otherwise you are going to end up with numbers like `0.14285714285714285`, no?

Reply:
you are totally right, thanks!